feat: migrates getting started information (#105)
posit-dev/connect-cookbook: index.qmd

---
title: Posit Connect Cookbook
---

The *Posit Connect Cookbook* contains recipes for Python and R using the *Posit SDK*, which you can utilize to accomplish everyday tasks on your Connect server. The recipes are lightweight and intend to be straightforward.

# [Getting Started](./01-getting-started/)

An overview of the Posit SDK and how to setup an authenticated client.

# [Groups](./03-groups/)

> Groups are utlized to to associate multiple users together and provide a simpler mechanism for managing content access controls.

Reviewer: ```suggestion
Groups consist of multiple users, and provide a streamlined way to manage content access controls.
```

Some typo fixes and light rewording
